libs/agent-scripts/cloudbroker/upgrade_cluster.py

```diff
@@ -22,15 +22,16 @@ def action():
     """
     zero_complexity_path = '/opt/code/github/0-complexity/'
     repo_path = zero_complexity_path + 'openvcloud_installer'
-
-    if j.system.fs.exists(repo_path):
-        j.system.process.execute('cd %s && git reset --hard' % repo_path)
-        j.system.process.execute('cd %s && git checkout master' % repo_path)
-        j.system.process.execute('cd %s && git pull' % repo_path)
-    else:
-        j.system.process.execute(
-            'cd %s && git clone https://github.com/0-complexity/openvcloud_installer.git' % zero_complexity_path)
-
+    scl = j.clients.osis.getNamespace('system')
+    versionmodel = scl.version.searchOne({'status': 'INSTALLING'})
+    manifest = yaml.load(versionmodel['manifest'])
+    for repo in manifest['repos']:
+        if repo['name'] == 'OpenVcloud Installer':
```

Comment:
i wouldnt write OpenVcloud but instead OpenvCloud that said, I rather have you matching against the url then the name